charts/argocd-apps/values.yaml

```diff
@@ -1,253 +1,256 @@
+#                  url: https://1.2.3.4
+#                - cluster: engineering-prod
+#                  url: https://2.4.6.8
+#                - cluster: finance-preprod
+#                  url: https://9.8.7.6
+#    template:
+#      apiVersion: argoproj.io/v1alpha1
+#      kind: ApplicationSet
+#      metadata:
+#        name: "{{ .name }}"
+#      spec:
+#        generators: *generators
+#        template:
+#          metadata:
+#            name: "{{`{{cluster}}`}}-guestbook"
+#          spec:
+#            project: my-project
+#            source:
+#              repoURL: https://github.com/infra-team/cluster-deployments.git
+#              targetRevision: HEAD
+#              path: guestbook/{{`{{cluster}}`}}
+#            destination:
+#              server: "{{`{{cluster}}`}}"
+#              namespace: guestbook
+#  second:
+#    items:
+#      - name: my-appset
+#        generators:
+#          - list:
+#              elements:
+#                - cluster: engineering-dev
+#                  url: https://1.2.3.4
+#                - cluster: engineering-prod
+#                  url: https://2.4.6.8
+#                - cluster: finance-preprod
+#                  url: https://9.8.7.6
+#    template: |-
+#      apiVersion: argoproj.io/v1alpha1
+#      kind: ApplicationSet
+#      metadata:
+#        name: {{ .name }}
+#      spec:
+#        generators: {{ toYaml .generators | nindent 4 }}
+#        template:
+#          metadata:
+#            name: '{{`{{cluster}}`}}-guestbook'
+#          spec:
+#            project: my-project
+#            source:
+#              repoURL: https://github.com/infra-team/cluster-deployments.git
+#              targetRevision: HEAD
+#              path: guestbook/{{`{{cluster}}`}}
+#            destination:
+#              server: '{{`{{cluster}}`}}'
+#              namespace: guestbook
 
 # -- DEPRECATED - Please refer [Deprecation Notice](https://github.com/argoproj-labs/argocd-extensions?tab=readme-ov-file#deprecation-notice) for more info.
 # @default -- `[]` (See [values.yaml])
 ## This function in tech preview stage, do expect unstability or breaking changes in newer versions. Bump image.tag if necessary.
 ## Ref: https://github.com/argoproj-labs/argocd-extensions
-extensions: []
-  # - name: example
-  #   namespace: argocd
-  #   additionalLabels: {}
-  #   additionalAnnotations: {}
-  #   sources:
-  #     - git:
-  #         url: https://github.com/argoproj-labs/argocd-example-extension.git
-  #     - web:
-  #         url: https://github.com/argoproj-labs/argocd-example-extension/releases/download/v0.1.0/extension.tar
+extensions: {}
+#  example:
+#    namespace: argocd
+#    additionalLabels: {}
+#    additionalAnnotations: {}
+#    sources:
+#      - git:
+#          url: https://github.com/argoproj-labs/argocd-example-extension.git
+#      - web:
+#          url: https://github.com/argoproj-labs/argocd-example-extension/releases/download/v0.1.0/extension.tar
+#
```

Comment:
Please fix the chart-test error due to wrong EOF

Reply:
@mkilchhofer done as requested